### DISCUSSION TITLE
经验条下方的红色条是什么模组 Whats the mod that shows a bar under xp bar

### DISCUSSION THREAD

**Discussion by zijialaope** [Rad586/Forget-Me-Not#83] [Q&A] [answered]
到下界那个增长的红条是哪个模组

**Comment by Rad586** [comment, marked as answer]
Kubejs。在 `advancement.js`（搜索文件名）删除这些行即可禁用。
Kubejs. Delete these lines in `advancement.js`(search file name) to remove this feature.
![image](https://github.com/user-attachments/assets/1cd3cbfa-1e26-40ff-a53a-f210009a7f7e)

**Reply by zijialaope** [reply]
谢谢帮助
